col/src/main/java/vct/col/resolve/Resolve.scala

```diff
@@ -82,6 +83,8 @@ case object ResolveTypes {
     case cls: Class[G] =>
       // PB: needs to be in ResolveTypes if we want to support method inheritance at some point.
       cls.supports.foreach(_.tryResolve(name => Spec.findClass(name, ctx).getOrElse(throw NoSuchNameError("class", name, cls))))
+    case _: JavaStringValue[G] =>
+      Java.findJavaTypeName(Java.JAVA_LANG_STRING, ctx)
```

Comment:
This line can be removed; the `t` member of JavaStringValue triggers this lookup